perfmetrics/scripts/continuous_test/ml_tests_tf/build.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Installing docker engine
+echo "Installing linux utility packages, like, lsb-release, curl..."
+sudo apt-get update
+sudo apt-get install -y \
+    ca-certificates \
+    curl \
+    gnupg \
+    lsb-release
+
+echo "Installing docker framework..."
+sudo mkdir -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+
+sudo apt-get update
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-compose-plugin -y
+
+cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse/"
+
+# Using the script from dlc_testing branch
+mkdir dlc_setup
+cd dlc_setup
+
+mkdir container_artifacts && mkdir container_artifacts/logs && mkdir container_artifacts/output
+
+# Building the dlc based dockerfile
+git clone https://github.com/GoogleCloudPlatform/gcsfuse.git
+cd gcsfuse
+git checkout dlc_testing
```

Comment:
is this because logging code is not yet merged? if yes please add that comment here

Reply:
Done